wolf-data/src/Wolf/Data/Export/Types.hs

```diff
@@ -89,3 +143,65 @@ instance ToJSON Repo where
             , "notes" .= repoNotes
             , "suggestions" .= repoSuggestions
             ]
+
+type Warn = [Problem]
```

Comment:
These are rather badly named.
I guess this should be `ExportWarning` and `ExportProblem` I can't tell, but I guess so.